ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/converters/EventStreamHttpConverter.java

```diff
@@ -61,7 +61,7 @@ public void write(@NotNull EventStreamTO eventStreamTO, MediaType contentType, H
 		Model eventStreamModel = eventStreamConverter.toModel(eventStreamTO);
 		Lang lang = rdfModelConverter.getLang(contentType, REST_ADMIN);
 		rdfModelConverter.checkLangForRelativeUrl(lang);
-		outputMessage.getHeaders().setContentType(contentType);
+		outputMessage.getHeaders().setContentType(MediaType.parseMediaType(lang.getHeaderString()));
```

Comment:
Why not using contentType anymore, as this was the original input and where the lang variable is created from